geowave-analytics/src/main/java/mil/nga/giat/geowave/analytics/tools/dbops/DirectBasicAccumuloOperationsBuilder.java

```diff
@@ -0,0 +1,60 @@
+package mil.nga.giat.geowave.analytics.tools.dbops;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import mil.nga.giat.geowave.accumulo.BasicAccumuloOperations;
+
+import org.apache.accumulo.core.client.AccumuloException;
+import org.apache.accumulo.core.client.AccumuloSecurityException;
+
+public class DirectBasicAccumuloOperationsBuilder implements
+		BasicAccumuloOperationsBuilder
+{
+	public static Map<String, BasicAccumuloOperations> connectionSet = new HashMap<String, BasicAccumuloOperations>();
```

Comment:
We do have an accumulo connector pool that caches connections already, not sure if its worth having another cache.